internal/pkg/aws/s3/s3.go

```diff
@@ -52,3 +61,64 @@ func (s *Service) PutArtifact(bucket, fileName string, data io.Reader) (string,
 
 	return resp.Location, nil
 }
+
+// EmptyBucket deletes all objects within the bucket.
+func (s *Service) EmptyBucket(bucket string) error {
+	var listResp *s3.ListObjectVersionsOutput
+	var err error
+	listParams := &s3.ListObjectVersionsInput{
+		Bucket: aws.String(bucket),
+	}
+	// Remove all versions of all objects.
+	for {
+		listResp, err = s.s3Svc.ListObjectVersions(listParams)
+		if err != nil {
+			return fmt.Errorf("list objects for bucket %s: %w", bucket, err)
+		}
+		if len(listResp.Versions) == 0 {
+			break
+		}
+		objectsID := make([]*s3.ObjectIdentifier, 0, batchSize)
```

Comment:
can we remove `batchSize` and just say: `make([]*s3.ObjectIdentifier, 0, len(listResp.Versions))`